### PR TITLE
fix: explicitly need to pull docker images, fixes #1525

### LINKED ISSUE #1525: ftl serve will not automatically download postgres:image
```sh
$ LOG_LEVEL=debug ftl serve
debug: Unary RPC failed: unavailable: dial tcp 127.0.0.1:8892: connect: connection refused: /xyz.block.ftl.v1.ControllerService/Ping
info: Starting FTL with 1 controller(s)
debug: Creating docker container 'ftl-db-1' for postgres db
ftl: error: failed to create db container: Error response from daemon: No such image: postgres:latest
```

### PR DESCRIPTION
**Pull request by gak**
The output is a bit messy but it at least will show progress, rather than hiding the output.

### CODE REVIEW

File: internal/container/container.go [hunk 1]
```diff
@@ -40,6 +40,27 @@ func DoesExist(ctx context.Context, name string) (bool, error) {
...
+		return err
+	}
+
+	reader, err := cli.ImagePull(ctx, image, types.ImagePullOptions{})
+	if err != nil {
+		return fmt.Errorf("failed to pull %s image: %w", image, err)
+	}
+	defer reader.Close()
+
+	_, err = io.Copy(os.Stderr, reader)
```
Comment (alecthomas) [inline internal/container/container.go:56]: Grab a logger from the context with `logger := log.FromContext(ctx)` and use `logger.WriterAt(log.Info)` to get a writer that will go through the logger.

**Review by alecthomas — APPROVED** [review]

File: internal/container/container.go (hunk 1 above)
Comment (alecthomas) [reply internal/container/container.go:56]: From memory the output is designed for a terminal though, but I think there's an option to modify it...maybe? Should be fine for now.

File: cmd/ftl/cmd_schema_import.go [hunk 2]
```diff
@@ -151,6 +151,11 @@ func (s *schemaImportCmd) setup(ctx context.Context) error {
 		}
 		_ = l.Close()
 
+		err = container.Pull(ctx, "ollama/ollama")
```
Comment (matt2e) [inline cmd/ftl/cmd_schema_import.go:154]: TIL we are using ollama

File: internal/container/container.go (hunk 1 above)
Comment (gak) [reply internal/container/container.go]: Works great, still messy but at least it's going through the logger.

**Pull request merged**